static/app/views/insights/sessions/queries/useCrashFreeSessions.tsx

```diff
@@ -35,7 +36,9 @@ export default function useCrashFreeSessions() {
       {
         query: {
           ...locationQuery.query,
-          interval: getSessionsInterval(datetime),
+          interval: pageFilters
+            ? getSessionsInterval(pageFilters.datetime)
+            : getSessionsInterval(datetime),
```

Comment:
l: idk, whichever is easier to read in your opinion


```suggestion
          interval: getSessionsInterval(pageFilters ? pageFilters.datetime : datetime),
```